workflow/rules/datavzrd.smk

```diff
@@ -78,3 +141,58 @@ rule go_enrichment_datavzrd:
         samples=get_model_samples,
     wrapper:
         "v3.13.8/utils/datavzrd"
+
+
+# Generating Meta Comparison Datavzrd Reports
+rule meta_compare_datavzrd:
+    input:
+        config=lambda wildcards: workflow.source_path(
+            f"../resources/datavzrd/meta_comparison-{wildcards.method}-template.yaml"
+        ),
+        table="results/tables/{method}/meta_compare_{meta_comp}.tsv",
+        plot="results/meta_comparison/{method}/{meta_comp}.json",
+    output:
+        report(
+            directory("results/datavzrd-reports/{method}_meta_comparison_{meta_comp}"),
+            htmlindex="index.html",
+            caption="../report/meta_compare.rst",
+            category="Comparisons",
+            subcategory="{meta_comp}",
+            patterns=["index.html"],
+            labels=lambda wildcards: get_meta_compare_labels(
+                method=f"{wildcards.method.capitalize()}: "
+            )(wildcards),
+        ),
+    params:
+        pathway_db=config["enrichment"]["spia"]["pathway_database"],
+        species=config["resources"]["ref"]["species"],
+    log:
+        "logs/datavzrd-report/meta_comp_{method}.{meta_comp}.log",
+    wrapper:
+        "v3.13.8/utils/datavzrd"
+
+
+# Generating Input Datavzrd Reports
+rule inputs_datavzrd:
+    input:
+        config=lambda wc: workflow.source_path(
+            f"../resources/datavzrd/{wc.input}-template.yaml"
+        ),
+        table=lambda wc: config[wc.input],
+    output:
+        report(
+            directory("results/datavzrd-reports/inputs/{input}"),
+            htmlindex="index.html",
+            # caption="../report/{input}.rst",
```

Comment:
Snakemake does not resolve the wildcard in the caption. I just left it out for now

Reply:
I removed even the comment because of linting